### PR TITLE
test: Allow test VMs have swap

### PR DESCRIPTION
**Pull request by jrajahalme**
Allow test VMs have swap configured. This does not change how the CI
VMs are deployed, but allows developer run local VMs with swap and
have CI k8s install not fail due to it.

Signed-off-by: Jarno Rajahalme <jarno@covalent.io>

### CODE REVIEW

**Comment by jrajahalme** [comment]
test-me-please

**Comment by jrajahalme** [comment]
retest-runtime

**Comment by jrajahalme** [comment]
retest-gke

**Review by aanm — APPROVED** [review]

**Comment by aanm** [comment]
@jrajahalme let's wait for @nebril review before merging.

**Comment by pchaigno** [comment]
Out of curiosity, when was this needed? Do we have a set of test configs not used in CI for which memory consumption is higher?

**Comment by jrajahalme** [comment]
> Out of curiosity, when was this needed? Do we have a set of test configs not used in CI for which memory consumption is higher?

Not needed in the CI (which has no swap configured), but when testing locally in a laptop that has only 16GB of RAM this makes it easier to run 2 VMs.

**Review by nebril — APPROVED** [review]

**Pull request merged**
